scratch.py

```diff
@@ -0,0 +1,6 @@
+import dask.bag
```

Comment:
shoudl this file be here?

Reply:
No! Good spot. Trigger happy on the `git commit -am`...